src/test/java/com/flightstats/hub/webhook/WebhookLeaderLocksTest.java

```diff
@@ -4,10 +4,7 @@
 import com.flightstats.hub.util.SafeZooKeeperUtils;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.curator.framework.CuratorFramework;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Minor nit: I think we had at one point decided never to collapse imports by namespace. It's the single customization I have to remember to make in Intellj.

Reply:
I have re-installed intellij and forgot to go in and fix that setting, thanks for catching that